mmscan/evaluator/vg_evaluation.py

```diff
@@ -0,0 +1,329 @@
+import numpy as np
+import torch
+from terminaltables import AsciiTable
+from tqdm import tqdm
+
+from mmscan.evaluator.metrics.box_metric import (compute_for_subset,
+                                                 get_average_precision,
+                                                 get_multi_topk_scores)
+from mmscan.utils.box_utils import index_box, to_9dof_box
+
+
+class VG_Evaluator:
+    """Evaluator for MMScan Visual Grounding benchmark.
+
+    Attributes:
+        eval_metric: All the evaluation metric, includes
```

Comment:
args are hard coded?
And the docstring does not match the default value in the code